### PR TITLE
What's new add experimental features

### PR DESCRIPTION
**Pull request by julenka**
## Overview
Adds several missing new experimental features to the "what's new in 2.3" page.

## Changes
- Fixes: #7241

### CODE REVIEW

File: Documentation/ReleaseNotes.md [hunk 1]
```diff
@@ -106,12 +106,35 @@ A hand physics extension service has been added to allow for using physics inter
...
+
+A keyboard that can be used on platforms which do not provide native keyboard support.
+([#6492](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6573))
+
+<img src="https://user-images.githubusercontent.com/168492/73916489-5b181d00-4872-11ea-9c1e-7ef6738a9f6f.png" width="400">
+
+**Hand coach (Experimental)**
+
+Hand animations that can give helpful hints for gestures users should perform.
+A ([#6493](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/1493))
```
Comment (wiwei) [inline Documentation/ReleaseNotes.md]:
```suggestion
([#6493](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/1493))
```

File: Documentation/ReleaseNotes.md [hunk 2]
```diff
@@ -106,12 +106,35 @@ A hand physics extension service has been added to allow for using physics inter
...
 The Pinch Slider has been updated to orient TrackVisuals, TickMarks and ThumbRoot based on the sliderAxis orientation ([#6858](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6858))
 
-![Y axis slier](https://user-images.githubusercontent.com/42405657/71687606-37a31380-2d96-11ea-84b5-ffe2368f8b57.JPG)
-![X axis slider](https://user-images.githubusercontent.com/42405657/71687640-4984b680-2d96-11ea-9f59-7732a91edd1b.JPG)
+<img src="https://user-images.githubusercontent.com/42405657/71687606-37a31380-2d96-11ea-84b5-ffe2368f8b57.JPG" width="150">
+<img src="https://user-images.githubusercontent.com/42405657/71687640-4984b680-2d96-11ea-9f59-7732a91edd1b.JPG?s=25" width="150">
+
+**ObjectManipulator and BoundsControl (Experimental)**
+
+ObjectManipulator and BoundsControl are refactored versions of ManipulationHandler and BoundingBox, respectively. They are designed to be simpler to configure, use and maintain. We are hoping to eventually deprecate ManipulationHandler and BoundingBox in favor of these more robust components.([#6294](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6924))
```
Comment (wiwei) [inline Documentation/ReleaseNotes.md]:
```suggestion
ObjectManipulator and BoundsControl are refactored versions of ManipulationHandler and BoundingBox, respectively. They are designed to be simpler to configure, use and maintain. We are hoping to eventually deprecate ManipulationHandler and BoundingBox in favor of these more robust components. ([#6294](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6924))
```

**Comment by davidkline-ms** [comment]
Thank you@julenka!

**Pull request merged**